cns/cns_util.go

```diff
@@ -49,9 +37,6 @@ func GetQuerySnapshotsTaskResult(ctx context.Context, taskInfo *vim25types.TaskI
 	}
 	if taskInfo.Result != nil {
 		snapshotQueryResult := taskInfo.Result.(cnstypes.CnsSnapshotQueryResult)
-		if &snapshotQueryResult == nil {
```

Comment:
I don't understand this.  Was this a pointer to a pointer?  If so, do we no longer want to check this?